helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml

```diff
@@ -106,6 +106,10 @@ spec:
     preKubeadmCommands:
       - bash /tmp/kubeadm/patches/kube-apiserver-patch.sh {{ .Values.controlPlane.resourceRatio }}
       - bash /run/kubeadm/gs-kube-proxy-patch.sh
```

Comment:
is this patch script running even if `$.Values.proxy.enabled` resolves to false?

I meant, should not be inside the conditional?